ska_sun/sun.py

```diff
@@ -34,7 +36,10 @@ def read_func(filename):
     return dat
 
 
-ROLL_TABLE = LazyVal(load_roll_table)
+def get_roll_table():
+    repo_dir = os.environ.get("CHANDRA_MODELS_REPO_DIR")
+    version = os.environ.get("CHANDRA_MODELS_DEFAULT_VERSION")
+    return _get_roll_table(repo_dir, version=version)
```

Comment:
If you change your version of SKA in the middle of this, all bets are off.  So that's still a weakness.

Reply:
I updated to remove the handling of CHANDRA_MODELS_REPO_DIR so I think this will now be OK with SKA changing too.